### ISSUE TITLE
SynthesizeTensors emits q/k/v biases unconditionally, would corrupt synthetic SmolLM weights

### ISSUE THREAD

**Issue by khurram-uworx** [khurram-uworx/Nivara#426]
SynthesizeTensors emits q/k/v biases unconditionally, would corrupt synthetic SmolLM weights

Found while planning the GPU kernel probe (khurram/gpu-probe).

`samples/NivaraInference/Qwen.cs` `SynthesizeTensors(LlamaConfig)` unconditionally adds
`q_proj`/`k_proj`/`v_proj` bias tensors (needed for the Qwen2 synthetic benchmark, where Qwen's
qkvBias inference engages). `samples/Nivara.Samples/LlamaForCausalLM.cs` `LlamaLoader.Load`
infers `qkvBias` from tensor *presence*:

    bool qkvBias = tensors.ContainsKey("model.layers.0.self_attn.q_proj.bias");

SmolLM-135M (`samples/data/smollm-135m/config.json`) sets `attention_bias: false` — the real
checkpoint has no q/k/v bias tensors. Generating synthetic SmolLM weights via
`SynthesizeTensors(SmolLMConfig)` would silently produce `qkvBias == true`, so the synthetic
model's attention math (logits) would differ numerically from the real checkpoint — invalidating
any synthetic SmolLM benchmark (decode parity, A/B, GPU-vs-CPU).

Not urgent: no synthetic SmolLM benchmark exists yet, and the GPU probe uses its own fixture
generator. Fix direction:
- Gate bias emission on the config (`attention_bias`) — add `AttentionBias` to `LlamaConfig`
  parsed from `config.json`, or parameterize `SynthesizeTensors(config, includeBias)`; or
- Have `LlamaLoader` prefer an explicit config flag over presence inference.

Acceptance: `SynthesizeTensors(SmolLMConfig)` emits no bias tensors; `SynthesizeTensors(QwenConfig)`
still emits them; both load with the correct `qkvBias`.
